content/pl_docker/docker_course_dev.md

```diff
@@ -173,3 +173,49 @@ When developing locally on PrairieLearn, you **must** click "Load from Disk" eve
 
 That's it!
 You're ready to develop questions for the OPB!
+
+# Setting up local autograder container
+For development of questions that require an autograder, you will need to set up a local autograder container as well. 
+
+## Important! (Windows Only)
+If you are using Windows, the following commands will need to be executed in Windows Subsystem for Linux (WSL). You can do this two ways:
+1. Open a command prompt and type `wsl` to open a WSL terminal
+1. Look for the Windows Subsystem for Linux application in the Start Menu
+<img src="pl_images/wsl_icon_windows.png">
```

Comment:
I don't fully understand here, what do you mean the "following commands" - do you mean "setting up your local Docker instance for auto grading will need to be done on WSL?"

Reply:
Yes - on Windows for pl autograding to work you have to run docker through a wsl2 terminal instance, because of how docker works on windows (if you use the hyper-v version of windows docker backend it might be different, but its discouraged by docker to use that and it requires windows 10/11 pro (or enterprise maybe?) to use)